htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -121,3 +121,36 @@ ALTER TABLE llx_societe ADD INDEX idx_societe_warehouse(fk_warehouse);
 
 ALTER TABLE llx_socpeople MODIFY poste varchar(255);
 
+
+create table llx_paiementtva
+(
+  rowid           integer AUTO_INCREMENT PRIMARY KEY,
+  fk_tva          integer,
+  datec           datetime,           -- date de creation
+  tms             timestamp,
+  datep           datetime,           -- payment date
+  amount          double(24,8) DEFAULT 0,
+  fk_typepaiement integer NOT NULL,
+  num_paiement    varchar(50),
+  note            text,
+  fk_bank         integer NOT NULL,
+  fk_user_creat   integer,            -- creation user
+  fk_user_modif   integer             -- last modification user
+
+)ENGINE=innodb;
+
+ALTER TABLE llx_tva ADD paye smallint default 0 NOT NULL;
+ALTER TABLE llx_tva ADD fk_account integer;
+
+INSERT INTO llx_paiementtva (fk_tva, datec, datep, amount, fk_typepaiement, num_paiement, note, fk_bank, fk_user_creat, fk_user_modif)
```

Comment:
For new table, we must try to use english version. Can you use instead
llx_payment_vat
to match llx_payment_salary or llx_payment _various